### PR TITLE
Updating T2.1 Images

### PR DESCRIPTION
**Pull request by Harikrishnan-MSFT**
Sample list - 

**app-anonymous-users
app-complete-auth
bot-archive-groupchat-messages
bot-release-management
bot-Sharepoint-List
bot-task-module
graph-appcatalog-lifecycle
incoming-webhook
meeting-details-tab
meeting-live-code-interview
meeting-recruitment-app
meeting-tabs
msgext-link-unfurling-meeting
Msgext-Message-reminder
msgext-search-sso-config
tab-Staggered-Permissions
bot-proactive-messaging-teamsfx -  https://github.com/OfficeDev/Microsoft-Teams-Samples/issues/1311

### CODE REVIEW

File: samples/app-anonymous-users/nodejs/Images/anonymoususersupport.gif
Comment (Pawank-MSFT) [inline samples/app-anonymous-users/nodejs/Images/anonymoususersupport.gif:1]: In Gif why we are showing invalid facebook id image?

File: samples/app-complete-auth/csharp/AppCompleteAuth/Images/app-complete-auth.gif
Comment (Pawank-MSFT) [inline samples/app-complete-auth/csharp/AppCompleteAuth/Images/app-complete-auth.gif:1]: Hide email

File: samples/app-complete-auth/csharp/AppCompleteAuth/Images/bot-user.png
Comment (Pawank-MSFT) [inline samples/app-complete-auth/csharp/AppCompleteAuth/Images/bot-user.png:1]: Hide email from image check everywhere

File: samples/app-complete-auth/csharp/AppCompleteAuth/Images/tab-sso.png
Comment (Pawank-MSFT) [inline samples/app-complete-auth/csharp/AppCompleteAuth/Images/tab-sso.png:1]: hide email

File: samples/tab-staggered-permission/csharp/StaggeredPermission/Images/user-info.png
Comment (Pawank-MSFT) [inline samples/tab-staggered-permission/csharp/StaggeredPermission/Images/user-info.png:1]: hide email

File: samples/tab-staggered-permission/csharp/StaggeredPermission/Images/user-mails.png
Comment (Pawank-MSFT) [inline samples/tab-staggered-permission/csharp/StaggeredPermission/Images/user-mails.png:1]: Hide email

**Review by Pawank-MSFT — COMMENTED** [review]
given some comments please fix

**Review by Pawank-MSFT — APPROVED** [review]
Looks correct, Approving!

**Review by Mohammed-MSFT — APPROVED** [review]

**Pull request merged**